superset/dashboards/filter_state/commands/get.py

```diff
@@ -26,11 +26,9 @@
 class GetFilterStateCommand(GetKeyValueCommand):
     def get(self, resource_id: int, key: str, refresh_timeout: bool) -> Optional[str]:
         dashboard = DashboardDAO.get_by_id_or_slug(str(resource_id))
-        if dashboard:
-            entry: Entry = cache_manager.filter_state_cache.get(
-                cache_key(resource_id, key)
-            )
-            if refresh_timeout:
-                cache_manager.filter_state_cache.set(key, entry)
-            return entry["value"]
-        return None
+        entry: Entry = cache_manager.filter_state_cache.get(
+            cache_key(resource_id, key)
+        ) or {}
+        if dashboard and entry and refresh_timeout:
+            cache_manager.filter_state_cache.set(key, entry)
```

Comment:
I'm assuming `dashboard` will always be truthy if we've gotten this far, so maybe just 
```suggestion
        if entry and refresh_timeout:
            cache_manager.filter_state_cache.set(key, entry)
```